lib/models/reminder.dart

```diff
@@ -1,18 +1,10 @@
 
 class Reminder {
 
-  String title;
-  String description;
+  final int id;
+  final String title;
+  final String description;
 
-  Reminder(this.title, this.description);
-
-}
-
-class EditableReminder {
-
-  final Reminder reminder;
-  final int index;
-
-  EditableReminder(this.reminder, this.index);
+  Reminder(this.title, this.description, [this.id]);
```

Comment:
O que significam os colchetes do `[this.id]`?

Reply:
No Dart, eles identificam parâmetros opcionais